test/test-helpers/misc.h

```diff
@@ -33,6 +33,10 @@ std::vector<std::string> file_contents(const std::string& filepath);
  */
 bool starts_with(const std::string& prefix, const std::string& input);
 
+/* \brief Returns `true` if `input` ends with `suffix`.
+ */
+bool ends_with(const std::string& input, const std::string& suffix);
```

Comment:
The parameters are now asymmetric in the header (but not in the implementation). Other than that, I personally prefer the original parameter order, probably because that's what I usually come across. I reckon the input string could be seen as the self parameter.

Reply:
Thanks! I fixed the order for `ends_with` in 2744fed.

> I personally prefer the original parameter order

Hmm. I'll wait for one more vote to break the tie :)